docs/graphs/Prims_Algorithm.md

```diff
@@ -0,0 +1,74 @@
+# Prim's Algorithm for Minimum Spanning Tree
+
+## Overview
+Prim's Algorithm is a greedy algorithm that finds the Minimum Spanning Tree (MST) for a connected weighted graph. It efficiently connects all vertices with the minimum total edge weight, making it a fundamental tool in various applications such as network design and clustering.
+
+## Table of Contents
+- [Introduction](#introduction)
+- [How Prim's Algorithm Works](#how-prims-algorithm-works)
+- [Step-by-Step Execution](#step-by-step-execution)
+- [Time Complexity](#time-complexity)
+- [Applications](#applications)
+- [Comparison with Kruskal's Algorithm](#comparison-with-kruskals-algorithm)
+- [Conclusion](#conclusion)
+
```

Comment:
delete the table of contents, we have default setup for this in sidebar

Reply:
Okay I will do in the format which u have quoted I will delete this part